.github/workflows/deploy-node-aws.yml

```diff
@@ -30,3 +30,9 @@ jobs:
         env:
           REGISTRY_URL: ${{ secrets.AWS_NODE_REGISTRY_URL }}
           PACKAGE_NAME: ironfish
+
+      - name: Push version string to API
+        run: ./ironfish-cli/scripts/push-version.sh
+        env:
+          IRON_FISH_API_KEY: ${{ secrets.IRON_FISH_API_KEY }}
+          IRON_FISH_API_URL: ${{ secrets.IRON_FISH_API_URL }}
```

Comment:
I think we might not want it as part of this workflow, because sometimes we'll deploy test Docker images to AWS just for our API to use that we aren't ready to release publicly. The `deploy-node-github` one could be okay though -- the GitHub docker registry is only used by users as far as I know.

I don't mind having it as a separate workflow step to add to our deploy steps for now either. Another idea could be to have it run when we create a new release in GitHub -- not sure if that's a good idea or not